.github/workflows/release.yml

```diff
@@ -0,0 +1,102 @@
+name: release
+
+on:
+  push:
+    tags: 
+      - '*.*.*'
```

Comment:
I think we should change this because we might want to have releases with version numbers such as: `x.y.z` or `x.y.z-something`.
The rest of the PR looks ok to me.

Reply:
@sduenas, I have updated the PR. Please let me know what do you think.